pkg/balancer-js/src/utils/errors.ts

```diff
@@ -55,6 +55,7 @@ const balancerErrorCodes: Record<string, string> = {
   '327': 'SWAPS_DISABLED',
   '328': 'CALLER_IS_NOT_LBP_OWNER',
   '329': 'PRICE_RATE_OVERFLOW',
+  '330': 'INVALID_JOIN_EXIT_WHILE_PAUSED',
```

Comment:
```suggestion
  '330': 'INVALID_JOIN_EXIT_KIND_WHILE_SWAPS_DISABLED',
```
It is important to not get 'paused' (an emergency mechanism that can only be used the first 3 months) and 'swaps disabled' (a manager action).